packages/api-db-karma/src/model.ts

```diff
@@ -166,7 +166,10 @@ export const EmbedBlockModel = struct({
   url: optional(string),
   title: optional(string),
   width: optional(int32),
-  height: optional(int32)
+  height: optional(int32),
```

Comment:
the `api-db-karma` database adapter is currently not maintained

Reply:
reverted